src/main/minio.js

```diff
@@ -30,12 +30,12 @@ import mkdirp from 'mkdirp'
 import path from 'path'
 import _ from 'lodash'
 
-import { isValidPrefix, isValidEndpoint, isValidBucketName,
+import { extractMetadata, prependXAMZMeta, isValidPrefix, isValidEndpoint, isValidBucketName,
   isValidPort, isValidObjectName, isAmazonEndpoint, getScope,
   uriEscape, uriResourceEscape, isBoolean, isFunction, isNumber,
   isString, isObject, isArray, pipesetup,
   readableStream, isReadableStream, isVirtualHostStyle,
-  probeContentType, makeDateLong, promisify } from './helpers.js'
```

Comment:
we might still have to use `probeContentType` to check on cases where `contentType` is not provided as part of the metadata

Reply:
Discussed